core/src/main/java/apoc/cypher/CypherInitializer.java

```diff
@@ -46,23 +52,31 @@ public void available() {
         Util.newDaemonThread(() -> {
 
             try {
-                awaitApocProceduresRegistered();
+                final boolean isSystemDatabase = db.databaseName().equals(GraphDatabaseSettings.SYSTEM_DATABASE_NAME);
+                if (!isSystemDatabase) {
+                    awaitApocProceduresRegistered();
+                }
                 Configuration config = dependencyResolver.resolveDependency(ApocConfig.class).getConfig();
 
-                TreeMap<String, String> initializers = Iterators.stream(config.getKeys(ApocConfig.APOC_CONFIG_INITIALIZER_CYPHER))
-                        .collect(Collectors.toMap(k -> k, k -> config.getString(k),
-                                (v1, v2) -> {
-                                    throw new RuntimeException(String.format("Duplicate key for values %s and %s", v1, v2));
```

Comment:
does this break the server startup? if so, do we want that? or rather juts log an error?